sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -279,6 +330,9 @@ object Factorial {
   )
 }
 
+@ExpressionDescription(
+  usage = "_FUNC_(value) - Returns Factorial of the number",
```

Comment:
specify the valid range here?

Also might be more useful to say

Returns the n factorial. n is valid in [0, 20] range.